packages/astro/src/integration/namespace.ts

```diff
@@ -0,0 +1,26 @@
+import module from 'module';
+import type { Plugin } from 'vite';
```

Comment:
Is `vite` a dependency of the `@sentry/astro` package already? If not, we either must add it or get rid of this type somehow.

Reply:
Vite is a `dev` dependency. Should we still get rid of it?

Reply:
I think it's worth keeping it. There will probably the need for vite specific code at some point while developing the integration

Reply:
I added Vite as a dev dependency for types (originally also for the `loadEnv` functionality but encountered problems with using it there). If it doesn't cause us problems, let's keep it around.